DataTransferObject/UserDataTransferObject.php

```diff
@@ -2,10 +2,102 @@
 
 namespace SumoCoders\FrameworkMultiUserBundle\DataTransferObject;
 
+use SumoCoders\FrameworkMultiUserBundle\DataTransferObject\Interfaces\UserDataTransferObject
+    as UserDataTransferObjectInterface;
+use SumoCoders\FrameworkMultiUserBundle\User\Interfaces\User as UserInterface;
 use SumoCoders\FrameworkMultiUserBundle\User\User;
 
-interface UserDataTransferObject
+class UserDataTransferObject implements UserDataTransferObjectInterface
 {
-    public static function fromUser(User $user);
-    public function getEntity();
+    /**
+     * @var int
+     */
+    public $id;
+
+    /**
+     * @var string
+     */
+    public $userName;
+
+    /**
+     * @var string
+     */
+    public $displayName;
+
+    /**
+     * @var string
+     */
+    public $email;
+
+    /**
+     * @var User
+     */
+    private $user;
+
+    /**
+     * @param UserInterface $user
+     *
+     * @return self
+     */
+    public static function fromUser(UserInterface $user)
+    {
+        $baseUserTransferObject = new self();
```

Comment:
goh, misschien. Kvind het wel niet echt nodig en in alle andere DTO's is het ook zo